Vagrantfile

```diff
@@ -14,17 +14,34 @@ Vagrant.configure(2) do |config|
 
   config.vm.provision "shell", inline: <<-SHELL
     sudo apt-get update
-    sudo apt-get install -y autoconf automake build-essential git libfreetype6-dev libpng12-dev libtool nmap pkg-config python-all-dev python-matplotlib python-nose python-numpy python-pip python-zmq sshpass trace-cmd
+    sudo apt-get install -y autoconf automake build-essential git libfreetype6-dev libpng12-dev libtool nmap pkg-config python-all-dev python-matplotlib python-nose python-numpy python-pip python-zmq sshpass trace-cmd openjdk-7-jdk openjdk-7-jre expect
     sudo pip install ipython[notebook] pandas
     sudo apt-get remove -y w3m
 
     ln -s /vagrant /home/vagrant/lisa
+
+    cd /home/vagrant/lisa
+    if [ ! -d ./tools/android-sdk-linux ]; then
+        wget -q https://dl.google.com/android/android-sdk_r24.4.1-linux.tgz
```

Comment:
Why the `-q`?  Showing some progress so that the user is not wondering why is this stalled is a good thing.  The last thing you can read before this is that w3m is being removed.  I would leave `wget`'s output, but if that's not possible or it's too noisy, then at least put an `echo Downloading Android SDK...` before executing `wget`.